docs/rootless.md

```diff
@@ -24,6 +24,12 @@ spec:
 
 See also the [example manifests](#Kubernetes).
 
+### Bottlerocket OS
+
+Needs to run `sysctl -w user.max_user_namespaces=N` (N=positive integer, like 63359) on the host nodes.
```

Comment:
Running this command in the Bottlerocket hosts isn't the recommended approach, I can update the docs to mention how to set up `max_user_namespaces` in the “Bottlerocket way”, if that's OK with you all.

Reply:
Yes, please

Reply:
Where is the documentation about this theme guys?